src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,620 @@
+# -*- coding: latin-1 -*-
+from collections import OrderedDict
+
+
+class _UnitsTable(object):
+    fundamental_units = {
+        "kg": "M",
+        "g": "M",
+        "lb": "M",
+        "lbm": "M",
+        "slug": "M",
+        "m": "L",
+        "cm": "L",
+        "ft": "L",
+        "in": "L",
+        "s": "T",
+        "A": "I",
+        "K": "?",
+        "C": "?",
+        "F": "?",
+        "R": "?",
+        "mol": "N",
+        "slugmol": "N",
+        "cd": "J",
+        "sr": "?",
+        "radian": "Angle",
+        "degree": "Angle",
+    }
+
+    derived_units = {
+        "N": "kg m s^-2",
+        "Pa": "N m^-2",
+        "W": "N m s^-1",
+        "J": "N m",
+        "V": "A ohm",
+        "F": "N m V^-2",
+        "H": "N m A^-2",
+        "S": "ohm^-1",
+        "Wb": "N m A^-1",
+        "T": "Wb m^-2",
+        "dyne": "g cm s^-2",
+        "erg": "dyne cm",
+        "pdl": "lbm ft s^-2",
+        "BTU": "J",
+        "psi": "lbf in^-2",
+        "lbf": "slug ft s^-2",
+        "psf": "lbf ft^-2",
+        "ohm": "kg m^2 s^-3 A^-2",
+        "Hz": "s^-1",
+    }
+
+    derived_units_with_conversion_factor = {
+        "l": (0.001, "m^3"),
+        "gal": (0.0037854117839999993, "m^3"),
+        "BTU": (1055.056, "J"),
+        "C": (lambda c: c + 273.15, "K"),
```

Comment:
as I discussed with @hpohekar, we don't need any specific functions for these: only generic functions based on conversion tables. The code will be updated.

Reply:
Code is updated and as per discussion, used this with different logic.